### ISSUE TITLE
Flickr Landscapes dataset release ?

### ISSUE THREAD

**Issue by Lokiiiiii** [NVlabs/SPADE#8]
Hi,
Will the Flickr Landscapes dataset be released ? If not could you give us the details of how the dataset was curated and pre-processed after being web scrapped ?

**Comment by mingyuliutw** [comment]
The Flickr dataset will not be released. The details of the curations that can be made public are given in the paper.

**Comment by mingyuliutw** [comment]
The Flickr dataset will not be released. The details of the curations that can be made public are given in the paper.

**Comment by GYengera** [comment]
Hello,

I had a question regarding which pretrained DeepLab model was used for computing segmentation maps for the Flickr landscape images. The DeepLab model trained on the Pascal VOC dataset appears to label most of the landscape images as background. So was a model that was pretrained on another dataset utilized or did you train a new semantic segmentation model on landscape images?
